descriptionCreator/modules/processData.js

```diff
@@ -183,11 +183,11 @@ export function processData(settings) {
           courseCredit: credits,
           courseLevel: level,
           courseExamCount: exams,
-          courseDirectory: `./exams/${courseCode}`
+          courseDirectory: `https://github.com/skipgu/past-exams/tree/main/exams/${courseCode}`
```

Comment:
Basically, the only change I did. This, however, in turn modified `exams.json` which is not desiable.